spaces/domains.json

```
@@ -18,4 +18,5 @@
   "dao.thirm.com": "thirm",
   "gov.synthetix.io": "synthetix",
   "voting.pancakeswap.finance": "pancake"
+  "signal.osho.finance": "love"
```

You missed a comma in the above line. This has been updated since your last commit, please pull it again before commit your changes. Thank you.
```suggestion
  "voting.pancakeswap.finance": "pancake",
  "signal.osho.finance": "love"
```